kie-spring-boot/jbpm-spring-boot/src/main/java/org/kie/springboot/jbpm/JBPMSBApplication.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.springboot.jbpm;
```

Comment:
IMO the package names for this should be org.jbpm.springboot and not org.kie.springboot.jbpm. 
@mswiderski wdyt here?

Reply:
+1 as the package name should reflect the artefact id as close as possible

Reply:
Done